tools/cldr-apps/src/main/java/org/unicode/cldr/web/UserRegistry.java

```diff
@@ -361,7 +361,11 @@ public boolean interestedIn(CLDRLocale locale) {
          */
         public LocaleSet getInterestLocales() {
             if (interestLocalesSet == null) {
-                interestLocalesSet = LocaleNormalizer.setFromStringQuietly(intlocs, null);
+                if (userIsManagerOrStronger(this)) {
+                    interestLocalesSet = LocaleNormalizer.setFromStringQuietly(intlocs, null);
+                } else {
+                    interestLocalesSet = LocaleNormalizer.setFromStringQuietly(locales, null);
+                }
```

Comment:
> The meaning of "interest locales" really needs clarification.

💯 exactly. part of the technical debt here.  "technical writing debt"?

> Seemingly the concept is only applicable to managers or stronger, and this method is being changed to implement a fallback to "authorized locales" for less-than-managers, right? 

The mismatch here is between the concept of "interest locales" (as in, _what I should be notified on_ ) vs. the DB field `cldr_users.intlocs`.  Prior to the previous PR, this function did return `locales.split()` for less-than-managers, and `intlocs.split()` for managers-or-stronger.

> Do all the callers take that into account appropriately?

This is the bottleneck function for getting a user's interest locales. The other call sites should all be here within the UserRegistry.

Reply:
> It might be better to call userCanSetInterestLocales instead of calling userIsManagerOrStronger directly here, to encapsulate the the "manager" dependency:

👍  this is a great suggestion. 

Basically:
1. for users who _can_ set their interest locales, the `intlocs` field contains the locales they are "interested in".
2. for users who _cannot_ set their interest locales, the `locales` field contains the locales they are "interested in" by virtue of being those assigned to that user.

(As I noted very recently, vetters who are assigned to "all locales" (if there is such a thing) are/were probably left in the lurch, having interest locales hard coded to nothing, and no way to set interest locales. We might want to explicitly remove this 'feature' in production, it's probably only useful for smoketest/development vetters who get allowed to vote 'everywhere'.)

Reply:
"The other call sites should all be here within the UserRegistry" -- `getInterestLocales` is also called from VotingParticipation and SurveyVettingParticipation. For less-than-managers, that will be the same as getting the authorized locales, so the distinction only applies to managers-or-stronger. For VotingParticipation and SurveyVettingParticipation, for managers-or-stronger, it's the interest locales rather than the authorized locales that are taken into account. I'm not sure why, or what the implications are, but I don't see a problem with it.